rmw/include/rmw/rmw.h

```diff
@@ -251,32 +256,6 @@ rmw_publisher_count_matched_subscriptions(
   const rmw_publisher_t * publisher,
   size_t * subscription_count);
 
-/// Retrieve the actual qos settings of the publisher.
```

Comment:
This is from https://github.com/ros2/rmw/commit/16d3cf4aaf3e964459b816aa32b0ff69bb95a477. We don't need to include this right now.